lsdcGui.py

```diff
@@ -354,14 +354,14 @@ def __init__(self, parent = None, **kwargs):
     def getSpotNodeList(self):
       nodeList = []
       for i in range (0,self.spotNodeCount):
-        nodeList.append(int(getBlConfig("spotNode"+str(i+1)).split('-')[1]))
+        nodeList.append(int(getBlConfig("spotNode"+str(i+1)).split('-')[1].split('cpu')[1]))
```

Comment:
Can you just drop the ".split('-')[1]" here? I think it would make the intent more clear, but I'm not sure if "cpu" would appear earlier in the string.

Reply:
that is a good point, at least as long as they don't make "cpu" part of the first part of the name of  a computing node.

Reply:
see commit 46341a6 below for the recommended fix (tested on beamline)